webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -72,6 +73,10 @@ const (
 	invalidMinHardwareVersionDowngrade       = "cannot downgrade hardware version"
 	invalidMinHardwareVersionPowerState      = "cannot upgrade hardware version unless powered off"
 	invalidImageKind                         = "supported: " + vmiKind + ", " + cvmiKind
+	invalidHostOrDomainName                  = "value must adhere to RFC-1123"
```

Comment:
@bryanv The `invalidHost...` constants are no longer necessary in this file, but I will hold off on removing them until you are finished with your review.